### ISSUE TITLE
My inventory moves whenever i try to move or shortcut drop items out of my inventory

### ISSUE THREAD

**Issue by Shayminato** [Wynncraft/Issues#6844]
# Delete this line if you've followed the README instructions and searched for a duplicate issue FIRST, and before posting any new issue.

**Comment by LordHelmchen98** [comment]
Well first off the title is not supposed to describe your issue in detail and scondly can you please describe your issue a little bit better? How exactly does it move?

**Comment by Shayminato** [comment]
Like Sometimes when i click on items, the invitory shifts left or right, or
when i use the hotkey Q to drop stuff, sorry i didn't know how it worked,
it's my first time using something like that

On Dec 5, 2016 3:08 PM, "Lord Helmchen" <notifications@github.com> wrote:

Well first off the title is not supposed to describe your issue in detail
and scondly can you please describe your issue a little bit better? How
exactly does it move?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Wynncraft/Issues/issues/6844#issuecomment-264993695>,
or mute the thread

**Comment by Shayminato** [comment]
Like Sometimes when i click on items, the invitory shifts left or right, or
when i use the hotkey Q to drop stuff, sorry i didn't know how it worked,
it's my first time using something like that

**Comment by LordHelmchen98** [comment]
Does this by any chance happen when you get potion effects or if potion effects run out?

**Comment by naybelline** [comment]
Mine does this too,  nearly thrown my important stuff out when grinding.
No potion effect, just constantly happens for me 
It's kinda annoying
I'm on 1.8.9

**Comment by Moe-Ronickah** [comment]
I'm still wondering about this.  Is this still going on?  Do your items look oversized and a bit fuzzy?  There is an error that can happen with the resource pack that makes everything go to a lower resolution, and you may be clicking the wrong box.

**Comment by naybelline** [comment]
Yeah it's still happening for me, everything looks ok, it's just the inventory moves side to side

**Comment by LordHelmchen98** [comment]
@Moe-Ronickah I'm pretty sure it's not an issue with the resource pack. It only happens it 1.8 but not in higher versions. I don't know the reason why, but if you click stuff sometimes you get effects that run out right after you got them causing your inventory to move from left to right and vice versa.

**Comment by Moe-Ronickah** [comment]
Can anyone get me a screenshot of this, please?

**Comment by LordHelmchen98** [comment]
I recorded it now, but I have to go. I'll add a youtuber link later.

**Comment by LordHelmchen98** [comment]
Alright here is a video of that glitch https://youtu.be/BQ4eV163xSE

**Comment by LordHelmchen98** [comment]
It seems to be related to weapons which are retextured half slabs
